### DISCUSSION TITLE
Comma 3 displays LKas fault code on 2017 Chrysler Pacifica

### DISCUSSION THREAD

**Discussion by Fritzie5** [community/community#40706] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Installed a comma 3 on my 2017 Chrysler Pacifica recently and it keeps giving me a lkas fault code when I start driving. I am new to these platforms and have no idea how any of this works but I’m having a real difficult time getting info on what I need to do to fix the problem. Any feedback or help would be greatly appreciated!

**Comment by ossanna16** [comment]
Hi there @Fritzie5 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
